src/hct_mis_api/apps/payment/admin.py

```diff
@@ -660,16 +660,20 @@ def has_add_permission(self, request: HttpRequest) -> bool:
 
 
 @admin.register(Account)
-class DeliveryMechanismDataAdmin(HOPEModelAdminBase):
-    list_display = ("individual", "get_business_area", "get_program", "account_type", "is_unique")
+class AccountAdmin(HOPEModelAdminBase):
+    list_display = ("individual", "number", "get_business_area", "get_program", "account_type", "is_unique")
 
     raw_id_fields = ("account_type", "individual")
     readonly_fields = ("unique_key", "signature_hash")
-    search_fields = ("individual__unicef_id",)
+    search_fields = (
```

Comment:
If you 're already on it, some optimization needs to be done, now search results in timeout

https://hope.unicef.org/api/unicorn/payment/account/?q=IND-22-0264.9339